lib/shared/image_preview.dart

```diff
@@ -25,6 +26,7 @@ class ImagePreview extends StatefulWidget {
   final void Function()? navigateToPost;
   final bool? isComment;
   final bool? read;
+  final bool showPlaceholder;
```

Comment:
I think we should always show a placeholder for images - we might not need to add this as an optional parameter. Thoughts?

Reply:
Sure! I wasn't sure if it would have a negative impact on any other places that `ImagePreview` is used, but I agree always having a placeholder makes sense.